gui/dialogs/c172p-oil-160.xml

```diff
@@ -84,6 +88,22 @@
         </group>
     </group>
     
+     <group>
+        <layout>hbox</layout>
```

Comment:
And since you're already fixing the last pieces of wrong indentation, you might add a space to this piece of code as well...

Reply:
or remove a space, in fact 😛